src/style.scss

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+.card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__icon {
+    display: flex;
+    padding: 32px 19px 40px;
+    img {
+      min-width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
```

Comment:
I saw that you setting horizontal margins for almost all card elements, but it is better to give padding for the card instead of it, and after it you just need to give bottom/top margins for elements